run/test/specs/disappearing_link.spec.ts

```diff
@@ -7,21 +7,22 @@ import { newContact } from './utils/create_contact';
 import { SupportedPlatformsType, openAppTwoDevices, closeApp } from './utils/open_app';
 import { setDisappearingMessage } from './utils/set_disappearing_messages';
 
-iosIt('Disappearing link message 1o1', disappearingLinkMessage1o1Ios);
-androidIt('Disappearing link message 1o1', disappearingLinkMessage1o1Android);
+iosIt('Disappearing link message 1:1', disappearingLinkMessage1o1Ios);
+androidIt('Disappearing link message 1:1', disappearingLinkMessage1o1Android);
+
+const time: DMTimeOption = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

Comment:
```suggestion
const time = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

I'll stop marking those, just search and replace

Reply:
Updated